src/test.py

```diff
@@ -42,12 +45,12 @@ def example_graph(self):
 
     def test_graph_factory(self):
         base_graph = self.example_graph()
-        assert base_graph == Graph(self.factory_input), \
+        assert repr(base_graph) == Graph(self.factory_input), \
             ('\n' + repr(base_graph) + '\n' + str(self.factory_input))
         g_double = Graph(eval(str(base_graph)))
         # WARN: Never compare two string literals: could be order sensitive, one object must be Graph
         #str(g_double) == str(base_graph)
-        assert g_double == base_graph, repr(g_double) + '\n' + repr(base_graph)
+        assert repr(g_double) + '\n' + repr(base_graph)
```

Comment:
This assert doesn't test anything.

Reply:
I misunderstood how assert statement works.